packages/tpetra/core/src/Tpetra_MultiVectorFiller.hpp

```diff
@@ -68,9 +69,12 @@ namespace Details {
   // newEntries into allEntries and make the results unique.  (This is
   // cheaper than sorting the whole array.)
   //
+  // \warning DO NOT CALL THIS METHOD!  This method is DEPRECATED
```

Comment:
If we deprecate the helper methods, then MultiVectorFiller itself will generate spurious build warnings.  Best practice is only to deprecate user-facing stuff.

Reply:
Easy enough to fix.